src/components/Bookstate.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export default function Bookstate() {
```

Comment:
- [ ] As mentioned by the previous reviewer, only one of the requirements has been met so far.

- [ ] I would kindly suggest that you write the logic and the markup for each component according to the project requirements. For example, the `<Book>` component should take the book data as a prop and render the title and the author, as well as a button for removing books. 


Here's an example of how you can write the <Book> component:

```jsx
import React from 'react';

function Book(props) {
  // Destructure the book data and the removeBook method from the props
  const { id, title, author, removeBook } = props;

  // A handler function to call the removeBook method with the book id
  const handleRemove = () => {
    removeBook(id);
  };

  return (
    <div className="book">
      {/* Render the title and the author of the book */}
      <h3>{title}</h3>
      <p>{author}</p>
      {/* Render a button for removing books */}
      <button onClick={handleRemove}>Remove</button>
    </div>
  );
}

export default Book;
```

Please let me know if you have any questions or feedback. Happy coding!👩‍💻
```

Please let me know if you have any questions or feedback. Happy coding!👩‍💻